### DISCUSSION TITLE
whether bfe support TLS1.3 or do we have plan about this

### DISCUSSION THREAD

**Discussion by wyaow** [bfenetworks/bfe#1138] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by iyangsj** [comment]
Duplicate of #79

**Comment by iyangsj** [comment]
TLS1.3 will be supported in later versions (v1.4 or v1.5)

**Comment by zhangbo1882** [comment]
Why do not use openssl/BoringSSL ?
